Microsoft.Toolkit.Uwp.SampleApp/SamplePages/SlidableListItem/SlidableListItemPage.xaml

```diff
@@ -27,7 +28,8 @@
                                        IsRightSwipeEnabled="{Binding DataContext.IsRightSwipeEnabled.Value, ElementName=Root, Mode=OneWay}"
                                        IsLeftSwipeEnabled="{Binding DataContext.IsLeftSwipeEnabled.Value, ElementName=Root, Mode=OneWay}"
                                        LeftCommand="{x:Bind ToggleFavorite}"
-                                       RightCommandRequested="SlidableListItem_RightCommandActivated"
+                                       RightCommand="{Binding DeleteItem, ElementName=Page, Mode=OneWay}"
```

Comment:
Just use x:Bind here rather than binding, means you don't have to name the page.